011-quarkus-panache-rest-flyway/src/main/resources/db/migration/V1.0.0__init.sql

```diff
@@ -1,4 +1,5 @@
-CREATE SEQUENCE hibernate_sequence START 1;
+CREATE SEQUENCE application_SEQ START 1 INCREMENT 50;
```

Comment:
@pjgg @michalvavrik why increment by 50?

This feels almost like a bug. What's the reason for increment by 50?

Reply:
it is not a bug, we can't use hibernate sequence and each table needs to have it's own sequence. I didn't argue with increment (though I don't know reason) as it strongly depends on allocation policy and pool strategy

Reply:
What I mean is that Hibernate Schema has sequence increment 50, so we can't use 1. But we could define allocation on the entity.

Reply:
https://jakarta.ee/specifications/persistence/2.2/apidocs/javax/persistence/sequencegenerator#allocationSize() says Default:  50 

That's probably the case.

What's strange to me is that increment definition wasn't needed before

Reply:
before we weren't using our sequence, but hibernate one. increment by 50 is a good thing in real life app; f.e. here in the first comment, I found explanation for pros of 50 https://discourse.hibernate.org/t/generated-value-strategy-auto/6481